controllers/metallb_controller.go

```diff
@@ -160,112 +146,26 @@ func (r *MetalLBReconciler) SetupWithManager(mgr ctrl.Manager, bgpType string) e
 func (r *MetalLBReconciler) syncMetalLBResources(config *metallbv1beta1.MetalLB) error {
 	logger := r.Log.WithName("syncMetalLBResources")
 	logger.Info("Start")
-	data := render.MakeRenderData()
-
-	data.Data["SpeakerImage"] = os.Getenv("SPEAKER_IMAGE")
-	data.Data["ControllerImage"] = os.Getenv("CONTROLLER_IMAGE")
-	data.Data["FRRImage"] = os.Getenv("FRR_IMAGE")
-	data.Data["IsOpenShift"] = r.PlatformInfo.IsOpenShift()
-	data.Data["NameSpace"] = r.Namespace
-	data.Data["KubeRbacProxy"] = os.Getenv("KUBE_RBAC_PROXY_IMAGE")
-	data.Data["DeployKubeRbacProxies"] = os.Getenv("DEPLOY_KUBE_RBAC_PROXIES") == "true"
-
-	data.Data["LogLevel"] = metallbv1beta1.LogLevelInfo
-	if config.Spec.LogLevel != "" {
-		data.Data["LogLevel"] = config.Spec.LogLevel
-	}
-
-	var err error
-	data.Data["MetricsPort"], err = valueWithDefault("METRICS_PORT", 7472)
-	if err != nil {
-		return err
-	}
-	data.Data["MetricsPortHttps"], err = valueWithDefault("HTTPS_METRICS_PORT", 27472)
-	if err != nil {
-		return err
-	}
-	data.Data["FRRMetricsPort"], err = valueWithDefault("FRR_METRICS_PORT", 7473)
-	if err != nil {
-		return err
-	}
-	data.Data["FRRMetricsPortHttps"], err = valueWithDefault("FRR_HTTPS_METRICS_PORT", 27473)
-	if err != nil {
-		return err
-	}
-	data.Data["MLBindPort"], err = valueWithDefault("MEMBER_LIST_BIND_PORT", 7946)
+	objs, err := r.helm.GetManifests(config)
 	if err != nil {
 		return err
 	}
-
-	objs, err := render.RenderDir(ManifestPath, &data)
-	if err != nil {
-		logger.Error(err, "Fail to render config daemon manifests")
-		return err
-	}
-
-	// We shouldn't spam the api server trying to apply PodMonitors if the resource isn't installed.
-	deployPodMonitors := os.Getenv("DEPLOY_PODMONITORS") == "true"
-	if podMonitorAvailable(r.Client) && deployPodMonitors {
-		podmonitors, err := render.RenderDir(PodMonitorsPath, &data)
-		if err != nil {
-			logger.Error(err, "Fail to render PodMonitors manifests")
-			return err
-		}
-		objs = append(objs, podmonitors...)
-	} else {
-		logger.Info("PodMonitors Resource not available in the cluster. Will not try to apply them.")
-	}
-
 	for _, obj := range objs {
-		objNS := obj.GetNamespace()
-		if objNS != "" { // Avoid setting reference on a cluster-scoped resource.
-			if err := controllerutil.SetControllerReference(config, obj, r.Scheme); err != nil {
-				return errors.Wrapf(err, "Failed to set controller reference to %s %s", obj.GetNamespace(), obj.GetName())
-			}
+		objKind := obj.GetKind()
+		// Skip applying role and role binding object
```

Comment:
nit: "because with the operator these are being set outside, either in manifests or via the csv"